lib/gradeDialog/grade_dialog.dart

```diff
@@ -0,0 +1,190 @@
+                style: TextStyle(
+                  color: Color(0xFF1353CB),
+                  fontSize: 26.0,
+                  fontWeight: FontWeight.bold,
+                  fontFamily: 'Poppins',
+                ),
+                textAlign: TextAlign.center,
+                maxLines: 1,
+              ),
+            ),
+            Expanded(
+              child: Container(
+                margin: const EdgeInsets.fromLTRB(20.0, 20.0, 20.0, 2.0),
+                child: GridView.builder(
+                    gridDelegate:
+                        const SliverGridDelegateWithFixedCrossAxisCount(
+                      crossAxisCount: 3,
+                      mainAxisSpacing: 3,
+                      crossAxisSpacing: 3,
+                    ),
+                    itemCount: _list.length,
+                    itemBuilder: (context, index) {
+                      bool checked = (index == selectedIndex);
+                      bool lastIndex = (index == (_list.length - 1));
+                      return GestureDetector(
+                        onTap: () {
+                          setState(() {
+                            selectedIndex = index;
+                          });
+                        },
+                        child: Container(
+                          margin: const EdgeInsets.fromLTRB(7.0, 5.0, 7.0, 5.0),
+                          height:
+                              (MediaQuery.of(context).size.width * 0.3) * 0.83,
+                          width: MediaQuery.of(context).size.width * 0.3,
+                          decoration: BoxDecoration(
+                            border: Border.all(color: const Color(0xFFBFBFBF)),
+                            shape: BoxShape.rectangle,
+                            borderRadius: const BorderRadius.all(
+                              Radius.circular(15.0),
+                            ),
+                          ),
+                          child: SizedBox(
+                            height: (MediaQuery.of(context).size.width * 0.3) *
+                                0.83,
+                            width: MediaQuery.of(context).size.width * 0.3,
+                            child:
+                                Stack(fit: StackFit.expand, children: <Widget>[
+                              Card(
+                                margin: const EdgeInsets.all(3.0),
+                                color: checked
+                                    ? const Color(0xFF2DAD9D)
+                                    : const Color(0xFFFFF8C9),
+                                shape: const RoundedRectangleBorder(
+                                  borderRadius:
+                                      BorderRadius.all(Radius.circular(15.0)),
+                                ),
+                                child: Padding(
+                                  padding: const EdgeInsets.fromLTRB(
+                                      10.0, 20.0, 10.0, 10.0),
```

Comment:
Padding not required. Center the text to the container

Reply:
removed the padding

Reply:
wrapped with Center Widget